assisted-events-scrape/storage/cluster_events_storage.py

```diff
@@ -47,6 +52,21 @@ def store(self, component_versions, cluster, event_list):
             events = self.process_events(cluster_bash_data, event_list, event_names)
             self.store_events(events, only_new_events=False)
 
+    @classmethod
+    def _hash_user_name(cls, cluster: dict):
+        """mask user name with unique generated FNV-1a 128 bit id"""
+        if "user_name" not in cluster:
+            return
+
+        user_name = cluster.get("user_name")
+        del cluster["user_name"]
+
+        # empty username
+        if not user_name:
+            cluster["user_id"] = -1
+        else:
+            cluster["user_id"] = fnv.hash(str(user_name).encode(), algorithm=fnv.fnv_1a, bits=128)
```

Comment:
Why `FNV-1a` algorithm and not something provided by standard library for example?

Reply:
Fast than most hash function, and give pretty close results to cryptographic hash algorithm (without the key). 
If you have a better idea I'll be glad to hear :)

Reply:
I was just curious about the criteria taken into account to choose this algorithm in particular.
The only requirement in this case is avoiding collisions as much as possible and IIUC this algorithm should do a very good job at that.
Normally I would avoid adding extra third party dependency if it can be avoided, as performance is not really an issue here - but I'll leave this decision to you I don't really have a strong opinion on this.